cmd/influxd/launcher/launcher.go

```diff
@@ -122,7 +151,7 @@ func (m *Launcher) Shutdown(ctx context.Context) {
 	}
 
 	m.logger.Info("Stopping", zap.String("service", "query"))
-	if err := m.queryController.Shutdown(ctx); err != nil {
+	if err := m.queryController.Shutdown(ctx); err != nil && err != context.Canceled {
```

Comment:
Just checking that this is special cased because we expect err to potentially be `context.Canceled` and therefore there is no real error that was encountered during shutdown.

Reply:
@desa yeah, this was a drive-by bug fix I noticed.  This code was incorrectly reporting the context cancel as a failure.